lib/tasks/users.rake

```diff
@@ -135,6 +135,17 @@ GROUP BY p.user_id
   puts 'Done!', ''
 end
 
+desc "Disable 2fa for username."
+task "users:disable_2fa", [:username] => [:environment] do |_, args|
+  username = args[:username]
+
+  user = find_user(username)
+  id = user.id
+  UserSecondFactor.where(user_id: id).each(&:destroy!)
```

Comment:
Oops it looks this shouldn't destroy everything. We have two different methods now so we should only disabled totp?

Reply:
The description of the rake task might have to change here.

Reply:
I don't know. That's way beyond my pay grade. I based my PR on what @SamSaffron posted in the link below and assumed that it was a good way to turn of 2fa.

https://meta.discourse.org/t/admin-locked-out-of-2fa/92156/2?u=pfaffman

I think that a rake task to disable two factor authentication would be helpful to people who get locked out. 

It seems like you have already spent more time responding to my PR than it would have taken you to write it from scratch, and it's not clear that I can get it right on one more try, so I'll bow out.

Reply:
Hey, @tgxworld. NOW I see what you mean about totp! :-)

Sadly, the backup codes didn't exist before and I just left my phone in a Paris taxi. I just submitted a PR for how I think we need to deal with the totp issue. Sorry I didn't understand before.

https://github.com/discourse/discourse/pull/6259